Rethinking_2/Chp_06.ipynb

```diff
@@ -0,0 +1,3109 @@
+{
```

Comment:
<p>You can merge this cell and the one above and use <code>az.summary</code> with <code>kind="stats"</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p><code>arviz</code>  summary doesn't work when the married column is a boolean.</p>

Reply:
<p>Changing <code>married</code>  column to <code>int</code>  breaks the plot... Have bodged something together, but would be ideal if <code>arviz</code>  didn't break on a boolean column. The error message it threw was far from clear too.</p>

Reply:
This is a good point. Have you filed an issue on ArviZ? Please do, if not, and thanks!

Reply:
Done - arviz-devs/arviz#1178